docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -7,7 +7,7 @@
   - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
 - Open the `your-app-name` directory in VS Code.
 - Add a valid contextId (i.e. Project Id) and iModelId for your user to the IMJS_CONTEXT_ID and IMJS_IMODEL_ID variables in the .env file within the application's root directory.
-  - You can obtain these ids from the "Show Ids" column of your [registration dashboard](https://www.itwinjs.org/getting-started/registration-dashboard?tab=1)
+  - You can obtain these ids from the "Show Ids" column of your [registration dashboard]($docs/getting-started/registration-dashboard?tab=1)

```

Comment:
The query params do not work with $docs syntax

Reply:
Fixed